modules/util/source-formatter/src/main/java/com/liferay/source/formatter/parser/JavaClassParser.java

```diff
@@ -78,7 +78,7 @@ public static JavaClass parseJavaClass(String fileName, String content)
 			StringBundler.concat(
 				"\n(public\\s+)?(abstract\\s+)?(final\\s+)?@?",
 				"(class|enum|interface)\\s+", className,
-				"([<|\\s][^\\{]*)\\{"));
+				"([<|\\s]?[^\\{]*)\\{"));
```

Comment:
@michaelmdelcavalcanti, can you please clarify this is necessary? Which use case are you addressing with this change? Is the use case associated to the `\r`?